### ISSUE TITLE
Missing CONTRIBUTING.md file

### ISSUE THREAD

**Issue by TeamDman** [TeamDman/SuperFactoryManager#545]
Common guidance like starting from 1.19.2 etc should be persisted in a file that newcomers can easily grok
